_news/30-PAGE_workshop_annoucement.md

```diff
@@ -0,0 +1,6 @@
+---
+title: TWO WORKSHOPS WITH EXAMPLES IN PK-SIM AND MOBI® (24-25 JUNE 2024 @PAGE)
```

Comment:
if using ® for MoBi - add it to PK-Sim as well

Reply:
> if using ® for MoBi - add it to PK-Sim as well

Done